uint8 motor_id

uint32 axis_state
uint32 control_mode
uint32 input_mode

float32 input_position
float32 input_velocity
float32 input_torque